examples/jsm/loaders/DRACOLoader.d.ts

```diff
@@ -0,0 +1,9 @@
+import {
+  LoadingManager
+} from '../../../src/Three';
+
+export class DRACOLoader {
+  constructor(manager?: LoadingManager);
+
```

Comment:
You should also add the actual public interface of the loader. So methods like `setPath()` or `setVerbosity()`.

Reply:
@Mugen87 How could I know which of these methods are meant to be public and which aren't? In the examples I've found, only a static `setDecoderPath` is used.

Reply:
You can see the public API in the docs:

https://threejs.org/docs/#examples/loaders/DRACOLoader

It looks like `setVerbosity` isn't documented, though.

Reply:
It can't hurt to provide typings for the entire public accessible interface.

Reply:
> It can't hurt to provide typings for the entire public accessible interface.

 Oh I agree -- it's just not necessarily clear what's considered "public" if it's not in the docs.

Reply:
I would suggest skipping the following methods, at least –

- decodeDracoFileInternal
- addAttributeToGeometry
- convertDracoGeometryTo3JS
- getAttributeOptions